crates/bpf-filtering/src/config.rs

```diff
@@ -1,10 +1,79 @@
+use std::{collections::HashSet, str::FromStr};
+
 use bpf_common::Pid;
 use pulsar_core::pdk::{ConfigError, ModuleConfig};
 
-use crate::maps::{DEFAULT_CGROUP_RULES, DEFAULT_INTEREST, DEFAULT_RULES};
+use crate::maps::{DEFAULT_CGROUP_RULES, DEFAULT_CONTAINER_RULES, DEFAULT_INTEREST, DEFAULT_RULES};
 
 use super::maps::Image;
 
+#[derive(Clone, Debug, Eq, Hash, PartialEq)]
+#[repr(i32)]
+pub enum ContainerEngine {
+    Docker = 1,
+    Podman = 2,
+}
```

Comment:
as we have already discussed there is a similar `enum ContainerEngineKind` in `process monitor` doing the same job. it can be shared

Reply:
Good point. I moved the `ContainerEngineKind` enum to `bpf-common`, added the `All` variant and added `FromStr` implementation to it.